src/components/user/PersonalProfileEditForm.js

```diff
@@ -9,9 +9,29 @@ import {
   TextField,
   Button,
 } from '@mui/material';
-import { Avatar, Alert } from '@mui/material';
+import { Alert } from '@mui/material';
 import axiosInstance from '../../apis/axiosClient';
 import { useNavigate } from 'react-router-dom';
+
+const ProfileImage = ({ user }) => {
+  if (user && user.profileImage && user.profileImage.url) {
+    return (
+      <img
+        src={user.profileImage.url}
```

Comment:
For firebase/google cloud storage, the URLs are served from "edge nodes" (essentially google has servers all over the world so they can serve files to users super-fast by using the server that is closest to that particular user). This is great but it also means that they do a lot of caching, so sometimes when we reload the page we won't see the image right away, and will still see the old image. 

In order to get around this, we can do something called "cache-busting" where we add a random string to the URL so that Google and our browser think this is a unique URL and serve us the fresh data. 

```suggestion
        src={user.profileImage.url + `?cachebust=${Math.random()}`}
```